website/docs/cli/oci_registries/provider-mirror.mdx

```diff
@@ -0,0 +1,310 @@
+---
+description: >-
+  Use OCI registries as an alternative installation source for OpenTofu providers
+---
+
+# Provider Mirrors in OCI Registries
+
+A "provider mirror" is a secondary location hosting a copy of a provider to be used instead
+of accessing the provider's primary OpenTofu Provider Registry. Creating a local mirror of some
+or all of the providers you use can reduce data transfer costs and can help with running
+OpenTofu in "air-gapped" environments that cannot access any services over the public internet.
+
+Alternative provider installation methods are configured as part of
+[the OpenTofu CLI Configuration](../config/config-file.mdx). You can configure installation
+from OCI Registries using an `oci_mirror` block as part of your
+[Explicit Installation Method Configuration](../config/config-file.mdx#explicit-installation-method-configuration).
+
+```hcl
+provider_installation {
+  oci_mirror {
+    repository_template = "example.com/opentofu-providers/${namespace}/${type}"
+    include             = ["registry.opentofu.org/*/*"]
+  }
+}
+```
+
+The above example specifies that any provider that belongs to the primary OpenTofu Registry
+should instead be installed from a repository in an OCI registry, constructing the
+repository address dynamically using the components of the provider source address.
+
+For example, the provider source address `hashicorp/tls` is a shorthand for
+`registry.opentofu.org/hashicorp/tls`, and so it matches the installation method rule
+configured above, with `hashicorp` as the "namespace" and `tls` as the "type".
+Therefore this configuration calls for the provider to be installed from the
+repository named `opentofu-providers/hashicorp/tls` in the OCI registry running at
+`example.com`.
+
+## `oci_mirror` installation method arguments
+
+- `repository_template`: A HCL-style template string that evaluates to an OCI repository
+  address to use for a given provider.
+
+    The following symbols are available for interpolation in the template:
+
+    - `hostname`: The hostname of the primary registry that the provider belongs to,
+      such as `registry.opentofu.org`.
+    - `namespace`: The namespace that the provider belongs to within its origin registry,
+      such as `hashicorp` in the above example.
+    - `type`: The provider's "type" name, which is the final portion of the source address
+      that's unique within a particular namespace, such as `tls` in the above example.
+
+    The template **must** include an interpolation for any component of the provider source
+    address that is not constrained exactly by the `include` argument. For example,
```

Comment:
`question`
And if setting `include = ["registry.opentofu.org/mycorp/*"]` then it means that the template should include interpolation only for the `type`?
If so, I will try to add just a small mention of it.

Reply:
Great points! Let's go with the current shape.